build/BundledVersions.targets

```diff
@@ -0,0 +1,72 @@
+<Project>
+  <PropertyGroup>
+    <BundledVersionsIntermediateDirectory>$(IntermediateDirectory)/GeneratedMSBuildImports</BundledVersionsIntermediateDirectory>
+    <BundledVersionsPropsFolder>15.0/Imports/Microsoft.Common.props/ImportBefore</BundledVersionsPropsFolder>
+    <BundledVersionsPropsFileName>Microsoft.NETCoreSdk.BundledVersions.props</BundledVersionsPropsFileName>
+  </PropertyGroup>
+
+  <Target Name="GenerateBundledVersionsProps"
+          BeforeTargets="PublishMSBuildExtensions">
+
+    <!-- 
+      The bundled version of Microsoft.NETCore.App is determined statically from DependencyVersions.props whereas the bundled version
+      of NETStandard.Library is defined by what Microsoft.NETCore.App pulls in. This digs in to the package resolution items of
+      of our build against Microsoft.NETCore.App to find the correct NETStandard.Library version
```

Comment:
Are we sure that there's a guarantee that the Microsoft.NETCore.App package will continue to have a dependency on NETStandard.Library?  I wouldn't be surprised if that goes away in 2.0.  Perhaps we should also just put the version of NETStandard.Library to use in DependencyVersions.props.